configs/rhino.config

```
@@ -0,0 +1,40 @@
+manifest {
+  name = 'autodiff-experiments'
+  author = 'Mathieu Fourment'
+  homePage = 'https://github.com/4ment/autodiff-experiments'
+  description = 'Benchmarking of automatic and analytical differentiation'
+  mainScript = 'main.nf'
+}
+
+profiles {
+    rhino {
+        process {
+            executor = 'slurm'
+            scratch = '/fh/scratch/delete30/matsen_e/mathieu/temp/nf-scratch/'
+            queue = 'campus-new'
+            cpus = 3
+            memory = 20.GB
+            container = 'quay.io/matsengrp/autodiff-experiments:rhino_config'
```

Please change it to 4ment/autodiff-experiments
```suggestions
container = '4ment/autodiff-experiments'
```

---

Hm, so for whatever reason your image was not working for me. Although it looks like it was re-build recently so I'll try again. 

however, a quay build using [your dockerfile](https://github.com/4ment/autodiff-experiments/blob/master/Dockerfile) - worked on my trial run.

I'll follow up here